packages/@pollyjs/core/src/persisters/rest/ajax.js

```diff
@@ -1,6 +1,6 @@
 const { keys } = Object;
-const REQUEST_ASYNC = !/PhantomJS/.test(self.navigator.userAgent);
-const NativeXMLHttpRequest = self.XMLHttpRequest;
+const REQUEST_ASYNC = !('navigator' in global) || !/PhantomJS/.test(global.navigator.userAgent);
```

Comment:
The assumption here was if dealing with node, whatever polyfill being used wouldn't have this nuance that was being guarded here